### DISCUSSION TITLE
Husky giving me hard times on a Dockerfile

### DISCUSSION THREAD

**Discussion by Pipazoul** [datatlas-erasme/front#103] [Q&A] [answered]
While trying to commit my new version of a Dockerfile the pre-commit integration stops me from commiting annoucing me 
``` bash
[91m[91m×[91m eslint --fix:[39m
[0m[0m
[0m[4mC:\path\to\datatlas\front\Dockerfile[24m[0m
[0m  [2m2:4[22m  [31merror[39m  Parsing error: Missing semicolon. (2:4)[0m
[0m[0m
[0m[31m[1m✖ 1 problem (1 error, 0 warnings)[22m[39m[0m
[0m[31m[1m[22m[39m[0m
husky - pre-commit hook exited with code 1 (error)
```

I've tried to ignore the docker file with an .eslintignore but no change

**Comment by lutangar** [comment, marked as answer]
`eslint` also runs prettier, try to add it to prettier ignore file :
https://prettier.io/docs/en/ignore.html#ignoring-files-prettierignore

But both tools aren't suppose to attempt parsing a `Dockerfile`...

If everything else fails, in last resort.... you may use `--no-verify` flag to avoid running the pre-commit hook.
You may create an issue and assign it to me, I'll take a look at it.

**Reply by Pipazoul** [reply]
The `--no-verify`  param works, but not the prettier ignore, ill open an issue
